app/src/main/java/com/sunasterisk/moviedb_51/utils/FragmentHelper.kt

```diff
@@ -0,0 +1,34 @@
+package com.sunasterisk.moviedb_51.utils
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentActivity
+import com.sunasterisk.moviedb_51.R
+
+fun FragmentActivity.replaceFragment(
+    id: Int,
+    fragment: Fragment,
+    addToBackStack: Boolean,
+    tag: String? = null,
```

Comment:
hai trường này thường dùng là false và null nên để giá trị mặc định là false, null 
```suggestion
    addToBackStack: Boolean = false,
    tag: String? = null,
```